src/main/java/peoplesoft/model/person/Person.java

```diff
@@ -150,8 +150,7 @@ public boolean equals(Object other) {
         }
 
         Person otherPerson = (Person) other;
-        return isSamePerson(otherPerson)
```

Comment:
tbh i'm not quite sure about this -- people typically expect `equals` to only return true if they're actually *equal* to each other (so in this case, it'd mean that every data field matches), but in this case the IDs may not be the same even though everything else matches

it might be better to have a separate method (uhh, maybe `isDuplicate()` or something?) that omits this ID check

Reply:
I agree with the separate method